webview-sample-plugin/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "webview-sample",
+  "description": "Cat Coding - A Webview API Sample",
+  "publisher": "theia",
+  "keywords": [
+    "theia-plugin"
+  ],
+  "version": "0.0.1",
+  "license": "none",
```

Comment:
Why licence is none?

Reply:
Good catch. It's from a plugin generator. Fixed.